sdks/python/apache_beam/pipeline.py

```diff
@@ -494,6 +494,9 @@ def replace_all(self, replacements):
     for override in replacements:
       self._check_replacement(override)
 
+  def _propagate_resource_hints(self):
+    ResourceHintsPropagator.propagate_hints_to_subtransforms(self)
```

Comment:
Nit: I'd go ahead and inline this.

Reply:
That was a hook for testing now removed with https://github.com/apache/beam/commit/5d58030998e77c066881891b94e4056c4778122a